solr/solr-ref-guide/modules/deployment-guide/pages/enabling-ssl.adoc

```diff
@@ -445,9 +445,9 @@ $ bin/solr post --solr-update-url https://localhost:8984/solr/mycollection/updat
 Use curl to query the SolrCloud collection created above, from a directory containing the PEM formatted certificate and key created above (e.g., `example/etc/`).
 If you have not enabled client authentication (system property `-Djetty.ssl.clientAuth=true)`, then you can remove the `-E solr-ssl.pem:secret` option:
 
-[source,bash]
+[source,console]
 ----
-curl -E solr-ssl.pem:secret --cacert solr-ssl.pem "https://localhost:8984/solr/mycollection/select?q=*:*"
+$ curl -E solr-ssl.pem:secret --cacert solr-ssl.pem "https://localhost:8984/solr/mycollection/select?q=*:*"
```

Comment:
Why the `$` here? It becomes part of the copied text when user clicks the copy button.

Reply:
Sorry, I only tested this in GitHub's renderer, which copied the `$`. Anotra will not https://docs.antora.org/antora-ui-default/code-blocks/#console-blocks

Reply:
;-).  One thing that I find annoying for me at least is that the copy only (for me) takes the first line..  I haven't figured out how to get it to copy multiple lines...  Have you seen the same behavior?  Maybe another bug/issue with antora?